src/shamli/Test1Program1.java

```diff
@@ -0,0 +1,38 @@
+package shamli;
+
+public class Test1Program1 
+{
+	int findCountOfChar(String name,char ch)
+	{
+		int count=0;
+		for(int i=0;i<name.length();i++)
+		{
+			if(name.charAt(i)==ch)
+			{
+				count++;
+			}
+		}
+		return count;
+	}
+	
+	void findFirstNonRecuringCharFromString(String input)
+	{
+		for(int index=0;index<input.length();index++)
+		{
+			int count=findCountOfChar(input,input.charAt(index));
+			if(count==1)
```

Comment:
here you can aslo use 
indexOf(ch) ==lastIndexOf(ch) logic to avoid one more for loop